packages/cw721/src/traits.rs

```diff
@@ -1,168 +0,0 @@
-use serde::de::DeserializeOwned;
-use serde::Serialize;
-
-use crate::{
-    AllNftInfoResponse, ApprovalResponse, ApprovalsResponse, ContractInfoResponse, NftInfoResponse,
-    NumTokensResponse, OperatorResponse, OperatorsResponse, OwnerOfResponse, TokensResponse,
-};
-use cosmwasm_std::{Binary, CustomMsg, Deps, DepsMut, Env, MessageInfo, Response, StdResult};
-use cw_utils::Expiration;
-
-pub trait Cw721<T, C>: Cw721Execute<T, C> + Cw721Query<T>
-where
-    T: Serialize + DeserializeOwned + Clone,
-    C: CustomMsg,
-{
-}
-
-pub trait Cw721Execute<T, C>
-where
-    T: Serialize + DeserializeOwned + Clone,
-    C: CustomMsg,
-{
-    type Err: ToString;
-
-    fn transfer_nft(
-        &self,
-        deps: DepsMut,
-        env: Env,
-        info: MessageInfo,
-        recipient: String,
-        token_id: String,
-    ) -> Result<Response<C>, Self::Err>;
```

Comment:
Before it e.g. `Cw721Execute` was rather an interface with no logic. Also note confusing generics naming like in `Cw721Execute<T, C>` and ...